rastervision/data/label_store/semantic_segmentation_raster_store.py

```diff
@@ -126,7 +128,11 @@ def save(self, labels):
                 class_mask = np.array(mask == class_id, dtype=np.uint8)
                 local_geojson_path = get_local_path(uri, self.tmp_dir)
 
-                transform = Affine.identity()
+                if isinstance(self.crs_transformer, RasterioCRSTransformer):
```

Comment:
If someone adds a new `CRSTransformer`, they might not remember to update this code which would end up returning an identity transform when that might not be appropriate.

Instead, you should add a method `get_affine_transform` to `CRSTransformer` which raises `NotImplementedError` as an exception. Then subclasses can implement their own way of generating and returning the affine transform.

Reply:
The reason I decided it shouldn't be an abstract method is that then subclasses would be forced to implement it, which seems like overkill since it's only used in this one place, and writers of new raster sources and crstransformer might not be interested in feature extraction.

Reply:
Okay.